flow/probes/ebpf.go

```diff
@@ -368,10 +81,13 @@ func (p *EBPFProbe) run() {
 	var startKTimeNs int64
 	var start time.Time
 
-	ebpfFlows := make(map[C.__u64]*EBPFFlow)
 	updateTicker := time.NewTicker(ebpfUpdate)
 	defer updateTicker.Stop()
 
+	kernFlows := [2001]C.struct_flow{}
```

Comment:
this size should be related to the chan size. I'm afraid that one day we will chan the chan size of this one without changing the other

Reply:
old comment :
replace 2001 by 
p.flow_pool_size = cap(flowChanOperation)+1

Reply:
done